cargo-shuttle/src/args.rs

```diff
@@ -430,6 +429,9 @@ pub struct LogsArgs {
     /// View all log lines
     #[arg(long, group = "output_mode")]
     pub all: bool,
+    /// Get logs from all deployments instead of one deployment
+    #[arg(long)]
+    pub dump_it_all: bool,
```

Comment:
Let's maybe use a more explicit name ? :p
```suggestion
    pub all_deployments: bool,
```